packages/components/label/src/label.tsx

```diff
@@ -75,15 +88,15 @@ const Label = (props: TLabelProps) => {
   );
 
   return (
-    <label id={props.id} htmlFor={props.htmlFor}>
-      <Text.Body tone={props.tone} isBold={props.isBold}>
+    <label css={getLabelStyles()} id={props.id} htmlFor={props.htmlFor}>
+      <TextComponent tone={props.tone} isBold={props.isBold}>
```

Comment:
I'm not sure if this is intended but now the `isBold` prop does not visually affect the component. It has `font-weight: 500` in both cases.

Reply:
Fixed in this commit - [9fab790](https://github.com/commercetools/ui-kit/pull/2356/commits/9fab79008b6f56b6e1f368aa32ab473ab39f55f0)